README.md

```diff
@@ -330,7 +330,42 @@ ctest -C RelWithDebInfo --parallel 8
 On Mac OSX using Xcode
 ======================
 
-#### Get the dependencies
+#### For Mac OSX 10.10 Yosemite and OS X 10.11 El Capitan
+Get **Xcode** from the App store. Open **Xcode** and *Agree* to license agreement. To *Agree* to to the license agreement, you may need to type in **Terminal**:
+```shell 
+sudo xcodebuild -license
+``` 
+
+Then, in **Terminal**, copy and paste commands below, line by line, one at a time. Be sure the output doesn't contain errors.
+```shell
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+brew install cmake swig
+brew cask install java
+git clone https://github.com/opensim-org/opensim-core.git
+mkdir opensim_dependencies_build
+cd opensim_dependencies_build
+cmake ../opensim-core/dependencies \
+      -DCMAKE_INSTALL_PREFIX="~/opensim_dependencies_install" \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
I first pasted them all in at once and there was too much output on terminal to tell whether there had been errors or not. Turns out there were, so I pasted them line by line and it went fine. I'll specify that the \ indicates that the command extends to the next line.